gradle/dependencies.gradle

```diff
@@ -124,7 +126,9 @@ ext {
             checkstyle   : "com.puppycrawl.tools:checkstyle:${pluginVersion.checkstyle}",
             firebase     : "com.google.firebase:firebase-plugins:${pluginVersion.firebase}",
             playPublisher: "com.github.triplet.gradle:play-publisher:${pluginVersion.gradlePlayPublisher}",
-            kotlin       : "org.jetbrains.kotlin:kotlin-gradle-plugin:${pluginVersion.kotlin}"
+            kotlin       : "org.jetbrains.kotlin:kotlin-gradle-plugin:${pluginVersion.kotlin}",
+            dependencyGraph : "com.vanniktech:gradle-dependency-graph-generator-plugin:${pluginVersion.dependencyGraph}",
```

Comment:
Minor minor thing: Could we reformat?

Reply:
Just pushed a commit to fix the spacing

![Screen Shot 2019-04-10 at 2 19 05 PM](https://user-images.githubusercontent.com/4394910/55914296-a2267a80-5b9b-11e9-8fa0-e36289fb1d95.png)